db/products.js

```
@@ -77,8 +77,56 @@ async function createProduct({
   }
 }
 
+async function destroyProduct({ id }) {
+  try {
+    const {
+      rows: [product],
+    } = await client.query(
+      `
+    DELETE FROM products 
+    WHERE id=$1
+    RETURNING *;
+    `,
+      [id]
+    );
+    return product;
+  } catch (err) {
+    console.error(err);
```

Let's throw these errors to the routes, which will send it to the front end